Management/DomainModels/Comment.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Common;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Management.DomainModels
+{
+    public class Comment
+    {
+        public Location Location { get; }
+        public UserId UserId { get; }
+        public string CommentStr { get; }
+        public DateTimeOffset CreatedAt { get; }
+        public Comment(Location location, UserId userId, string commentStr)
```

Comment:
I feel it might make more sense to still put everything in ctor since a `Comment` object is not considered fully constructed if some of the fields are missing.
I can update it to have a default `DateTimeOffset` parameter so that we only have one ctor.